README.md

```diff
@@ -79,6 +79,11 @@ To execute tests with grep tags on stage:
 $ npm run cy:run -- --env grepTags=@stage,url="BASE_URL_OF_APP",authorizationHeader="<SECRET HERE>"
 ```
 
+To only execute all.cy.js file which has all import test files
+```
+npm run cy:run -- cypress run --spec "cypress/e2e/all.cy.js", --env grepTags=@stage,url="https://apply-to-become-an-academy-internal-staging.london.cloudapps.digital",authorizationHeader="Rv3oKzr4a3ry3x"
```

Comment:
@Lisa-V-Sau we shouldn't really be putting this authorisation header into source control.
Please remove and replace with 'secret'

Reply:
oh dear, missed that... thanks